### PR TITLE
Add WLP-Platform to eeCompatible features

### LINKED ISSUE #28374: Versionless: Associate official platform strings with eeCompatible features
We don't want to burn in upstream platform detaills into the kernel layer, so associating the officual platform string names with corresponding eeCompatible features seems like a possible solution....

for example we could assign yet another feature header key value pair only in the compatibility features

com.ibm.websphere.appserver.eeCompatible-10.0.feature

would have WLP-Platform: jakartaee-10.0

### PR DESCRIPTION
**Pull request by jimblye**
fixes #28374

### CODE REVIEW

**Comment by jimblye** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.eeCompatible-9.0.feature [hunk 1]
```diff
@@ -7,3 +7,4 @@ Subsystem-Version: 9.0.0
 kind=ga
 edition=core
 WLP-Activation-Type: parallel
+WLP-Platform: jakartaee-9.0
```
Comment (rsherget) [inline dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.eeCompatible-9.0.feature]: Do we want this to be 9.0 or 9.1

Comment (cbridgha) [reply dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.eeCompatible-9.0.feature]: Agreed it needs  to match what you put for the features...  so jakartaee-9.1

Comment (jimblye) [reply dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.eeCompatible-9.0.feature]: done.   Should the file itself and the Subsystem-Version also be 9.1?

**Comment by jimblye** [comment]
#build

**Review by cbridgha — APPROVED** [review]
looks good

**Comment by cbridgha** [comment]
#libby

**Pull request merged**